package.json

```diff
@@ -5,12 +5,19 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node dist/server.js"
```

Comment:
Is there a build step missing somewhere? How about using [ts-node](https://www.npmjs.com/package/ts-node) and avoiding a build step?